src/classwork/04_assign/main.cpp

```diff
@@ -10,6 +12,28 @@ factorial.  Also, loop continues as long as user wants to.
 */
 int main() 
 {
+	int user_input = 0;
+	int answer = 0;
+	char choice;
+    do
+    {
+		cout<<"Please enter a number from 1-10 here: ";
+		cin>>user_input;
+		if (user_input >=1 && user_input <= 10)
+		{
+			answer = factorial(user_input);
+			cout<<"\n"<<"The factorial of "<< user_input<< " is: "<<answer<< "\n";
+		}
+		else
+		{
+			cout<<"Invalid answer.\n";
+		}
+		
```

Comment:
Must use a loop to validate the number is from 1 to 10.

while(user_input < 1 || user_input > 0)
{
   cout<<"Invalid number, enter a number from 1 to 10";
   cin>>user_input;
}